src/MessageRouter.py

```diff
@@ -0,0 +1,50 @@
+import time
+import re
+
+
+class MessageRouter:
+    """
+    Responsible for invoking a message command handler if a message contains
+    a command and the rate limit is not exceeded.
+    """
+
+    def __init__(self,
+                 command_handlers=None,
+                 default_rate_limit_in_seconds=60,
+                 per_command_rate_limit_in_seconds=None):
```

Comment:
As expressed in previous comment, we can simply do:

`per_command_rate_limit_in_seconds={}`